runtime/runtime/tests/runtime_group_tools/mod.rs

```diff
@@ -143,10 +142,16 @@ impl StandaloneRuntime {
         receipts: &[Receipt],
         transactions: &[SignedTransaction],
     ) -> (Vec<Receipt>, Vec<ExecutionOutcomeWithId>) {
+        // TODO - the shard id is correct but the shard version is hardcoded. It
```

Comment:
since the hardcoding is done by  the `fn default_shard_layout() -> ShardLayout { ShardLayout::single_shard() }` in `genesis_config.rs`, maybe also call that here? Like `let shard_layout = ShardLayout::single_shard(); let shard_uid = ...`. The TODO would still apply but the hackiness is sort of pulled a level up